plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray.yaml

```diff
@@ -0,0 +1,158 @@
+hydra:
+  launcher:
+    cls: hydra_plugins.hydra_ray_launcher.ray_launcher.RayLauncher
+    params:
+      app_config_path: /Users/jieru/workspace/hydra-fork/hydra/plugins/hydra_ray_launcher/example/conf
```

Comment:
Yes, I will look into if there's a better way to do this. 

I need the absolute path to conf because they also need to be rsynced  so hydra can find them when running on the cluster.